docs/monitoring/README.md

```diff
@@ -1,91 +1,106 @@
-# Prometheus Monitoring for TF operator
+# Prometheus Monitoring for TFJob
```

Comment:
@deepak-muley I think this doc may need some polishment later. As you owns that part, feel free to cut a PR when you get a change. I create an issue to track this update. https://github.com/kubeflow/training-operator/issues/1443

Reply:
Thank you, I agree!